app/src/main/res/layout/activity_main.xml

```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
+        android:layout_height="635dp"
         app:defaultNavHost="true"
         app:layout_constraintBottom_toTopOf="@id/nav_view"
+        app:layout_constraintEnd_toEndOf="parent"

```

Comment:
remove these fixed values

Reply:
end and start layout constraints are unnecessary when we have left and right layout constraints already

Reply:
Sorry, but how is this a fixed value?

Reply:
My bad, was copying and pasting ;P
See my comment after it